src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -83,7 +82,7 @@ public CursorsController(final CursorsService cursorsService,
         try {
             final List<SubscriptionCursor> cursors = cursorsService.getSubscriptionCursors(subscriptionId, client)
                     .stream()
-                    .map(cursor -> cursorConverter.convert(cursor, cursorTokenService.generateToken()))
+                    .map(cursor -> cursor.withToken(cursorTokenService.generateToken()))
```

Comment:
It would be clear if SubscriptionCursor has method for example `from(SubscriptionCursorWithoutToken, token)`